trestle/core/commands/author/ssp.py

```diff
@@ -123,46 +126,91 @@ def _init_arguments(self) -> None:
         self.add_argument('-m', '--markdown', help=file_help_str, required=True, type=str)
         output_help_str = 'Name of the output generated json SSP'
         self.add_argument('-o', '--output', help=output_help_str, required=True, type=str)
+        self.add_argument('-r', '--regenerate', action='store_true', help=const.HELP_REGENERATE)
+
+    def _imp_reqs_equivalent(
+        self, imp_reqs: List[ossp.ImplementedRequirement], orig_imp_reqs: List[ossp.ImplementedRequirement]
+    ) -> bool:
+        """
+        Check if imp_reqs are the same except for internal uuids.
+
+        Create copy of each imp_req and set the uuids within it to match the original - then check equality.
+        """
+        if len(as_list(imp_reqs)) == len(as_list(orig_imp_reqs)):
+            for reqs in zip(imp_reqs, orig_imp_reqs):
```

Comment:
If lists are None as expected above this will fail regardless, may be better to check if lists are none at the very beginning?

Reply:
I have removed as_list.